### PR TITLE
[test] RunTests VW options coverage script

### PR DESCRIPTION
**Pull request by lalo**
Script that generated this: https://github.com/VowpalWabbit/vowpal_wabbit/issues/2867

### CODE REVIEW

**Review by olgavrou — APPROVED** [review]
super duper nit, feel free to ignore: could we group the `BUT has default value` ones from the ones that are not tested at all?

File: test/runtest_coverage.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+from vowpalwabbit import pyvw
+
+def get_latests_tests():
```
Comment (jackgerrits) [inline test/runtest_coverage.py:3]:
```suggestion
def get_latest_tests():
```

File: test/runtests_parser.py [hunk 2]
```diff
@@ -234,12 +234,17 @@ def file_to_obj(filename):
 
     return results
 
-def main():
+def find_runtest_file():
+    rtfile = None
     possible_paths = ["./RunTests", "./test/RunTests"]
```
Comment (jackgerrits) [inline test/runtests_parser.py:239]: Consider making this not dependent on cwd?

Comment (lalo) [reply test/runtests_parser.py]: done

**Pull request merged**